### ISSUE TITLE
Implement isort `reverse_relative` rule

### ISSUE THREAD

**Issue by leroyvn** [charliermarsh/ruff#1813]
I'd love to move my import sorting to ruff; however I'm missing the "Reverse Relative" rule (see description here: https://pycqa.github.io/isort/docs/configuration/options.html#reverse-relative). Any chance to see it implemented in a foreseeable future?

**Comment by charliermarsh** [comment]
I inevitably have to read through the source to really understand these options :joy:

**Comment by charliermarsh** [comment]
Oh ok, does this just make it such that fewer dots come first? So it considers this correct?

```py
from . import a
from .. import a
from ... import a
```

**Comment by leroyvn** [comment]
Yes indeed, it reverses the sorting order of relative module imports and puts the closest to the top.

**Comment by charliermarsh** [comment]
Okay, cool, that's easy enough. Can probably go out in the next release later today.

**Comment by charliermarsh** [comment]
Will mark as `good first issue` in case someone wants to knock it out before I get to it!

**Comment by charliermarsh** [comment]
I may rename this to `relative-order` and accept `"closest-first"` and `"furthest-first"`. The current setting is a little confusing because it requires you to remember what the default order is (since "reverse" is only relative to the default). Any feedback on that?

**Comment by leroyvn** [comment]
I personally do not mind and agree with the terminology you are suggesting, but I think I'm biased.
